lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,24 @@ defmodule List do
   end
 
   @doc """
+  Returns true if `list` starts with `prefix`.
```

Comment:
I am wondering if `element` or `term` is a better name for the `prefix` argument